stage4.0_first_application/pycasa/ui/tasks/hello_world_pane.py

```diff
@@ -0,0 +1,33 @@
+from pyface.tasks.api import EditorAreaPane, IEditor, IEditorAreaPane, \
+    PaneItem, Task, TraitsTaskPane
+
+from traitsui.api import InstanceEditor, Item, ModelView, View
+from traits.api import HasStrictTraits, Instance, Str
+
+
+class HelloWorldModel(HasStrictTraits):
+
+    content = Str
+
+    def _content_default(self):
+        return "Hello World!"
+
+
+class HelloWorldView(ModelView):
+    model = Instance(HelloWorldModel, ())
+
+    def traits_view(self):
+        return View(
+            Item("model.content")
+        )
```

Comment:
Is the model and model view being redefined intentionally?

Reply:
No, meant to import. Removed.